index.js

```diff
@@ -150,9 +136,7 @@ S3Adapter.prototype.uploadFile = function (file, callback) {
 // - the file is set to a canned ACL (ie, headers:{ 'x-amz-acl': 'public-read' } )
 // - you pass credentials during your request for the file content itself
 S3Adapter.prototype.getFileURL = function (file) {
-	// Consider providing an option to use insecure http. I can't think of any
-	// sensible use case for plain http though. https should be used everywhere.
-	return this._knoxForFile(file).https(this._resolveFilename(file));
+	return 'https://' + (file.bucket || this.options.bucket) + '.s3.amazonaws.com' + this._resolveFilename(file);
```

Comment:
Thank you!

Maybe it would work to use `	return 'https://' + (file.bucket || this.options.bucket) + '.' + this.client.endpoint.host + this._resolveFilename(file);
`